test/app/AppKernel.php

```diff
@@ -41,6 +42,27 @@ public function registerContainerConfiguration(LoaderInterface $loader)
                     'resource' => '%kernel.root_dir%/config/routing.yml',
                 ),
             ));
+            $container->loadFromExtension('doctrine', array(
+                'dbal' => array(
+                    'driver' => 'pdo_mysql',
+                    'dbname' => 'api_test_case',
+                    'host' => 'localhost',
+                    'user' => 'root',
+                    'password' => 'toor',
```

Comment:
Never commit your local information! We should extract this to some external file with default values.